R/print.R

```diff
@@ -199,6 +198,7 @@ print_mcse_summary <- function(x, digits) {
     "Monte Carlo SE of elpd_loo is",
     paste0(.fr(mcse_val, digits), ".\n")
   )
+  print_reff_summary(x, digits)
```

Comment:
Right now `print_reff_summary` uses "MCSE" and `print_mcse_summary` uses "Monte Carlo SE". Should we use the same in both?

Reply:
I just forgot to push that commit